cmd/lambda/accounts/update_test.go

```diff
@@ -20,6 +20,11 @@ func ptrString(s string) *string {
 	return &ptrS
 }
 
+func ptr64(i int64) *int64 {
+	ptrI := i
+	return &ptrI
+}
+
 func TestUpdateAccountByID(t *testing.T) {
 
```

Comment:
Can you please add a test for 404- account not found

Reply:
This test doesn't go here and is covered here.  The account Service will either return the updated result or an error.  The error has the details on the coding.  

https://github.com/Optum/dce/blob/6346f4766c7cb05e32dfc4156638500875b171b8/pkg/data/account_test.go#L60-L69